File: programmers/pg_lv1_예산.js
function solution(d, budget) {
  let count = 0;

  d = d.sort((a, b) => a - b);
  for (i = 0; i < d.length; i++) {
    if (budget - d[i] >= 0) {
      budget = budget - d[i];
      count++;
    } else return count;
  }
  return count;
}

File: python/programmers/pg_lv1_예산.py
def solution(d, budget):
    d.sort()
    priceSum = 0
    count = 0
    
    for i in d:
        priceSum += i
        count += 1
        if priceSum > budget: return count - 1
        elif priceSum == budget: return count
    
    return count